.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -196,6 +196,7 @@ jobs:
       artifactName: drop
 
 - job: build_windows_2019
+  timeoutInMinutes: 120
```

Comment:
what's the additional build time because of the signing?

Reply:
around 12 mins increase